### PR TITLE
add elasticsearch-analysis-openkoreantext link

### PR DESCRIPTION
**Pull request by keepcosmos**
README에 elasticsearch-analysis-openkoreantext 추가했습니다.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,6 +97,7 @@ To include this in your Maven-based JVM project, add the following lines to your
 | Porting | [Python](https://github.com/open-korean-text/open-korean-text-python) | [Baeg-il Kim](https://github.com/cedar101) |
 | Package | [Python Korean NLP](https://github.com/konlpy/konlpy) | [KoNLPy](https://github.com/konlpy/konlpy) |
 | Package | [Elastic Search](https://github.com/open-korean-text/open-korean-text-elastic-search) | [socurites](https://github.com/socurites) |
+| Package | [Elastic Search](https://github.com/open-korean-text/elasticsearch-analysis-openkoreantext) | [open-korean-text](https://github.com/open-korean-text) |
```
Comment (nlpenguin) [inline README.md]: Add your ID instead of `[open-korean-text](https://github.com/open-korean-text)`

Comment (nlpenguin) [inline README.md]: I.e. `[keepcosmos](https://github.com/keepcosmos)`

**Comment by nlpenguin** [comment]
PTAL @keepcosmos

File: README.md (hunk 1 above)
Comment (keepcosmos) [inline README.md]: @nlpenguin openkoreantext-elasticsearch-plugin belongs to open-korean-text(organization), not my account.

Comment (nlpenguin) [inline README.md]: @keepcosmos It's the `contributor` column. I've updated it and merged. Please let me know if you want further changes!

**Pull request merged**